app/src/components/NetworkSettingsCard/__tests__/NetworkSettingsCard.test.js

```diff
@@ -22,4 +23,9 @@ describe('NetworkSettingsCard', () => {
     const wrapper = shallow(<NetworkSettingsCard />)
     expect(wrapper.exists(ClearDiscoveryCache)).toBe(true)
   })
+
+  it('should render a DisableDiscoveryCache component', () => {
+    const wrapper = shallow(<DisableDiscoveryCache />)
```

Comment:
Typo here! We want to make sure that rendering a `NetworkSettingsCard` ends up rendering a `DisableDiscoveryCache` as a child

```suggestion
    const wrapper = shallow(<NetworkSettingsCard />)
```